src/khepri_tx.erl

```diff
@@ -223,8 +223,14 @@ ensure_instruction_is_permitted({bs_add, _, _, _}) ->
     ok;
 ensure_instruction_is_permitted({bs_append, _, _, _, _, _, _, _, _}) ->
     ok;
+ensure_instruction_is_permitted({bs_private_append, _, _, _, _, _, _}) ->
+    ok;
+ensure_instruction_is_permitted(bs_init_writable) ->
+    ok;
 ensure_instruction_is_permitted({bs_init2, _, _, _, _, _, _}) ->
     ok;
+ensure_instruction_is_permitted({bs_init_bits, _, _, _, _, _, _}) ->
+    ok;
```

Comment:
Could you please keep the clauses in alphabetical order, based on the instruction name?